### PR TITLE
Updated all the endpoints

### PR DESCRIPTION
**Pull request by MehakKambo**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -119,40 +119,42 @@ def plant_conditions(scientific_name_raw: str):
...
+            illnesses = [dict(zip(response_keys, condition)) for condition in cursor.fetchall()]
     conn.close()
     return {
-        'conditions': conditions
+        'illnesses': illnesses
     }
 
-@app.route('/plants/<scientific_name_raw>/symptoms', methods=['GET'])
-def plant_symptoms(scientific_name_raw: str):
+@app.route('/plants/<illness_name_raw>/symptoms', methods=['GET'])
```
Comment (LucaOnline) [inline app.py:128]: Can we make this `/plants/<scientific_name_raw>/illness/<illness_name_raw>/symptoms` or something? It's a bit long, but having the parameter in interactions 4 and 5 mean different things depending on what comes after it is a bit confusing.

File: app.py [hunk 2]
```diff
@@ -161,65 +163,51 @@ def plant_illness_symptoms(plantIllness_name_raw: str):
...
     conn.close()
     return {
-        'handlingProtocol': handProtocol,
-        'symptoms': symptoms
+        'handlingProtocol': handProtocol
     }
 
-@app.route('/plants/<scientific_name_raw>,<common_name_raw>,<region_raw>/main', methods=['PUT'])
-def update_plant_info(scientific_name_raw: str, common_name_raw: str, region_raw: str):
+@app.route('/plants/<scientific_name_raw>,<common_name_raw>/main', methods=['PUT'])
```
Comment (LucaOnline) [inline app.py:182]: Is there a reason we're not allowing the plant region to be updated anymore?

File: app.py [hunk 3]
```diff
@@ -228,23 +216,69 @@ def add_plant_info(scientific_name_raw: str, common_name_raw: str, region_raw: s
...
+                    
     conn.close()
     return {
-        'ok': 'true'
-    }, 201
+        'ok': 'true',
+        'message': 'inserted successfully'
+    }
 
 @app.route('/plants/<symptom_name_raw>/plantIllness', methods=['GET'])
```
Comment (LucaOnline) [inline app.py:283]: See above comment on illnesses, we should either break this down to `/symptoms/<symptom_name_raw>/illnesses` or extend it to `/plant/<scientific_name_raw>/symptoms/<symptom_name_raw>/illnesses` for clarity.

**Review by LucaOnline — CHANGES_REQUESTED** [review]

**Review by LucaOnline — CHANGES_REQUESTED** [review]

File: app.py [hunk 4]
```diff
@@ -119,40 +119,47 @@ def plant_conditions(scientific_name_raw: str):
...
+                           'WHERE PI.name = %s AND P.scientificName = %s;', 
+                           (illness_name, scientific_name))
             symptoms = [dict(zip(response_keys, condition)) for condition in cursor.fetchall()]
     conn.close()
     return {
         'symptoms': symptoms
     }
 
-@app.route('/plants/<plantIllness_name_raw>/symptoms', methods=['GET'])
+@app.route('/plants/<plantIllness_name_raw>/handlingProtocol', methods=['GET'])
```
[the remark this answers is not in the document]
Comment (MehakKambo) [reply app.py]: Done

**Review by LucaOnline — APPROVED** [review]

**Pull request merged**